types/shared.d.ts

```diff
@@ -374,13 +374,37 @@ export interface IStripeConnectInitParams {
   locale?: string;
 }
 
+type ConnectHTMLElementRecordBase = {
```

Comment:
I'm seeing a compile time error in this branch, I think some code here still needs to be updated!

<img width="1140" alt="image" src="https://github.com/stripe/connect-js/assets/95381655/85bff677-3961-46f0-9e96-e6e921c8283b">

```

Reply:
Fixed this type issue; also should we update .github/workflows/test.yml to build for `beta` too?

Reply:
@jojo-stripe that would be good, I think that is a branch policy right?

Reply:
yes, let me add it to github

Reply:
https://github.com/stripe/connect-js/pull/66